### PR TITLE
wasm-check utility

### PR DESCRIPTION
**Pull request by NikVolf**
It just general check of compilation process (since there are number of toolchain/compiler errors might exist) 
1. Make sure that imported memory of the right size is used
2. Make sure no import outside parity runtime is used (for example, when std or other io library is accidently linked)

### CODE REVIEW

File: cli/check/main.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "coinbase",
+    "timestamp",
+    "difficulty",
+    "gaslimit",
+    "address",
+    "value",
+    "suicide",
+    "panic",
+    "elog",
+    "abort"
```
Comment (pepyakin) [inline cli/check/main.rs]: so you can't use `gas`, can you? : )

**Comment by pepyakin** [comment]
I'm not sure that I fully understand what this tool ought to check. It is both stricter (don't we allow the module to have no memories defined or imported?) and at the same time more lenient than the parity runtime (this tool checks for the names of the imported functions but doesn't check their signatures).

Aside from that, the code seems to be correct

File: cli/check/main.rs (hunk 1 above)
Comment (NikVolf) [reply cli/check/main.rs]: sure, because it supposed to check compiled contracts, which should not import gas manually

**Comment by NikVolf** [comment]
@pepyakin added some description

**Pull request merged**